cmd/snap/cmd_create_key.go

```diff
@@ -58,7 +55,7 @@ func (x *cmdCreateKey) Execute(args []string) error {
 	if keyName == "" {
 		keyName = "default"
 	}
-	if !validKeyName.MatchString(keyName) {
+	if !asserts.ValidAccountKeyName.MatchString(keyName) {
```

Comment:
Aha, so this is why it's exposed! Could we just rely on the error, and make sure the error's text is descriptive enough to cover this use case?